CMakeLists.txt

```diff
@@ -337,14 +337,18 @@ if(LOVR_USE_PICO)
 endif()
 
 if(LOVR_BUILD_SHARED)
+  list(REMOVE_ITEM LOVR_SRC src/main.c)
   add_library(lovr SHARED ${LOVR_SRC})
+  target_compile_definitions(lovr PRIVATE LOVR_BUILDING_SHARED)
```

Comment:
Same-ish comment as above -- why is the file getting omitted *and* a define getting added that accomplishes the same?  Maybe fallout from a previous change?

Reply:
That's because you're defining a WinMain in os_win32.c which needs to be ifdef'd out so it doesn't reference main when trying to build without a main

Reply:
I think I was confused and thought `int main` was getting ifdef'd out instead of the thing in os_win32.